docs/man/docker-attach.1.md

```diff
@@ -12,14 +12,14 @@ docker-attach - Attach to a running container
 CONTAINER
 
 # DESCRIPTION
-If you **docker run** a container in detached mode (**-d**), you can reattach to
-the detached container with **docker attach** using the container's ID or name.
+The **docker attach** command allows you to attach to a running container using
+the container's ID or name, either to view its ongoing output, or to control it
+interactively. You can attach to a container more than once.
```

Comment:
I think we need to better clarify what is meant by "attaching more than once." Right now it's not clear if we mean you can attach and detach repeatedly vs. having multiple attachments to a single container.